src/Core/Repositories/ISecretRepository.cs

```diff
@@ -6,8 +6,9 @@ public interface ISecretRepository
 {
     Task<IEnumerable<Secret>> GetManyByOrganizationIdAsync(Guid organizationId);
     Task<IEnumerable<Secret>> GetManyByIds(IEnumerable<Guid> ids);
+    Task<IEnumerable<Secret>> GetManyByProjectIdAsync(Guid projectId);
     Task<Secret> GetByIdAsync(Guid id);
-    Task<Secret> CreateAsync(Secret secret);
+    Task<Core.Entities.Secret> CreateAsync(Secret secret);
```

Comment:
We should just be able to have this as 

`Task<Secret> CreateAsync(Secret secret);`

Since we have the using statement `using Bit.Core.Entities;`

Reply:
Yes, thank you! Updated.